src/Tizen.AIAvatar/src/AIServices/OpenAIService.cs

```diff
@@ -0,0 +1,285 @@
+        /// </summary>
+        /// <param name="text">The text to be converted to speech.</param>
+        /// <param name="voice">Optional parameter to specify the voice type.</param>
+        /// <param name="options">Optional parameters for customizing speech output.</param>
+        /// <returns>A task representing the asynchronous operation, with a byte array of the generated audio data.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public async Task<byte[]> TextToSpeechAsync(
+            string text,
+            string voice = null,
+            Dictionary<string, object> options = null)
+        {
+            var client = ClientManager.GetClient(config.Endpoints.TextToSpeechEndpoint);
+
+            var request = new RestRequest("speech", Method.Post)
+                .AddHeader("Authorization", $"Bearer {config.ApiKey}")
+                .AddJsonBody(new
+                {
+                    model = "tts-1",
+                    input = text,
+                    voice = voice ?? "alloy",
+                    response_format = "mp3"
+                });
+
+            var response = await client.ExecuteAsync(request);
+            if (!response.IsSuccessful)
+                throw new Exception($"OpenAI TTS Error: {response.ErrorMessage}");
+
+            return response.RawBytes;
+        }
+
+        /// <summary>
+        /// Streams the given text as speech asynchronously.
+        /// </summary>
+        /// <param name="text">The text to be converted to speech.</param>
+        /// <param name="voice">Optional parameter to specify the voice type.</param>
+        /// <param name="options">Optional parameters for customizing speech output.</param>
+        /// <returns>A task representing the asynchronous operation.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public async Task TextToSpeechStreamAsync(string text, string voice = null, Dictionary<string, object> options = null)
+        {
+            const int SAMPLE_RATE = 24000;  // OpenAI TTS의 기본 샘플레이트
+            const int FRAME_DURATION_MS = 160;  // 160ms 단위로 분할
+            const int TAIL_DURATION_MS = 15;
+            const int BYTES_PER_SAMPLE = 2;  // 16-bit PCM
+            const int CHUNK_SIZE = (SAMPLE_RATE * FRAME_DURATION_MS * BYTES_PER_SAMPLE) / 1000;  // 160ms worth of PCM data
+            const int TAIL_CHUNK_SIZE = (SAMPLE_RATE * TAIL_DURATION_MS * BYTES_PER_SAMPLE) / 1000;
+
+            var client = ClientManager.GetClient(config.Endpoints.TextToSpeechEndpoint);
+            var request = new RestRequest("speech", Method.Post)
+                .AddHeader("Authorization", $"Bearer {config.ApiKey}")
+                .AddJsonBody(new
+                {
+                    model = "tts-1",
+                    input = text,
+                    voice = voice ?? "alloy",
+                    response_format = "pcm",  // PCM 포맷으로 변경
```

Comment:
Might be this Korean comment not be updated yet?

Reply:
I will remove the comments for clean code. I'll make the changes. 😄